client/src/app/site/pages/meetings/pages/motions/modules/motion-poll/components/motion-poll-detail-content/motion-poll-detail-content.component.ts

```diff
@@ -85,6 +88,27 @@ export class MotionPollDetailContentComponent extends BaseUiComponent implements
         super();
     }
 
+    ngDoCheck(): void {
+        let changeDetected = false;
+        const options = this._poll.options;
+        if (options.length !== this._oldOptions.length) {
+            changeDetected = true;
+        } else {
+            changeDetected = options.some((item, index) => {
+                return (
+                    item?.yes !== this._oldOptions[index]?.yes ||
+                    item?.no !== this._oldOptions[index]?.no ||
+                    item?.abstain !== this._oldOptions[index]?.abstain
+                );
+            });
+        }
+
+        if (changeDetected) {
+            this._oldOptions = JSON.parse(JSON.stringify(options));
+            this.setupTableData();
+        }
+    }
```

Comment:
Because I couldn't think of another way to find out in real time if the values in `this._poll.options` have changed and update the table data whenever it happens.

I know this is more than sub-optimal, so as I've already stated in my previous comment:
If you have other ideas I'd be glad to hear them ^^